<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Unreal Script Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="ui_run_all_btn">
           <property name="toolTip">
            <string>execute all commands</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ICONS:/executeAll.png</normaloff>ICONS:/executeAll.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_run_sel_btn">
           <property name="toolTip">
            <string>execute selected commands</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ICONS:/execute.png</normaloff>ICONS:/execute.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_clear_log_btn">
           <property name="toolTip">
            <string>clear unreal log</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ICONS:/clearHistory.png</normaloff>ICONS:/clearHistory.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_clear_script_btn">
           <property name="toolTip">
            <string>clear input script</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ICONS:/clearInput.png</normaloff>ICONS:/clearInput.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ui_clear_both_btn">
           <property name="toolTip">
            <string>clear all</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>ICONS:/clearAll.png</normaloff>ICONS:/clearAll.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>25</width>
             <height>25</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="ui_tab_widget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="ui_add_tab">
       <attribute name="title">
        <string>+</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Bahnschrift</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="ui_save_action"/>
    <addaction name="ui_open_action"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="ui_save_action">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="ui_open_action">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
